### DISCUSSION TITLE
Using per line comments (`//`) instead of block comments (`/* ... */`)

### DISCUSSION THREAD

**Discussion by the-shank** [LazyVim/LazyVim#2049] [Q&A] [answered]
How would I go about configuring LazyVim to use per line comments (`//`) instead of block comments (`/* ... */`). I understand that the comment functionality comes from `mini.comment`, but I couldn't find anything in their docs either about how to configure this. Can someone please point me in the right direction. Thanks!

**Comment by dpetka2001** [comment, marked as answer]
Take a look at this [comment](https://github.com/LazyVim/LazyVim/issues/1368#issuecomment-1735338404).

**Reply by the-shank** [reply]
I am sorry, somehow I missed that when I was searching for this. Thank you!
